float64[] xSrc
float64[] ySrc
---
float64[] xDst
float64[] yDst
